assets/panel/debugger.properties

```diff
@@ -440,6 +444,11 @@ welcome.search=%S to search for sources
 # a mac we use the unicode character.
 welcome.findInFiles=%S to find in files
 
+# LOCALIZATION NOTE (welcome.searchFunction): The center pane welcome panel's
+# search prompt. e.g. cmd+o to search for files. On windows, it's ctrl+shift+o, on
+# a mac we use the unicode character.
+welcome.searchFunction=%S to search for function in file
```

Comment:
I think this should either be "for a function" or "for functions"? (not a native English speaker though)